### ISSUE TITLE
Homebrew Formulae outdated

### ISSUE THREAD

**Issue by bradfol** [uber/needle#419]
The latest version of Needle currently available on Homebrew is v0.17.2, but it appears the latest release is v0.18.1

https://formulae.brew.sh/formula/needle
<img width="640" alt="Screen Shot 2022-06-07 at 7 22 47 PM" src="https://user-images.githubusercontent.com/620025/172517614-773d98ff-dd12-45bd-a2fa-71a7e3c12aa9.png">
.

**Comment by wbond** [comment]
I don't believe they will allow it to be updated at the moment due to usage of `lib_InternalSwiftSyntaxParser.dylib`. See https://github.com/Homebrew/homebrew-core/pull/99203 for more details.

**Comment by ikesyo** [comment]
#412 should be resolved for this.
